.pre-commit-config.yaml

```
@@ -37,31 +37,31 @@ repos:
     ]
 
 - repo: https://github.com/PyCQA/isort
-  rev: 5.13.2
+  rev: 6.0.1
   hooks:
   - id: isort
 
 - repo: https://github.com/psf/black-pre-commit-mirror
-  rev: 24.4.2
+  rev: 25.1.0
   hooks:
     - id: black
       language_version: python3
 
 - repo: https://github.com/asottile/blacken-docs
-  rev: 1.18.0
+  rev: 1.19.1
   hooks:
   - id: blacken-docs
     additional_dependencies: [black]
 
 - repo: https://github.com/codespell-project/codespell
-  rev: v2.3.0
+  rev: v2.4.1
   hooks:
   - id: codespell
     args: [-w]
     additional_dependencies: ["tomli; python_version<'3.11'"]
 
 - repo: https://github.com/PyCQA/flake8
-  rev: 7.1.1
+  rev: 7.2.0
   hooks:
   - id: flake8
     additional_dependencies: [flake8-bugbear>=23.2.13]
```

This one is also from #114 (thank you!). Should we add it here?

```suggestion
    additional_dependencies: [flake8-bugbear>=23.2.13]

- repo: https://github.com/abravalheri/validate-pyproject
  rev: v0.24.1
  hooks:
  - id: validate-pyproject
```